src/hooks/eventLogger.ts

```diff
@@ -63,6 +63,8 @@ export const useLogAllLinkClicks = (): void => {
 
 export const enum EventName {
     STATIC_VIDEO_PLAYED = 'StaticVideoPlayed',
+    ENROLLMENT = 'Enrollment',
```

Comment:
```suggestion
    ENROLLMENT = 'ABTestEnrolled',
```

Reply:
Do you plan to use the EventName.ENROLLMENT event name for future A/B tests as well? Or new event names for every test? If the latter, please rename this to reflect this specific test. Perhaps something like `ABTestEnrolled:<test name>`